pages/platform/private-registry/using-private-registry-with-kubernetes/guide.de-de.md

```diff
@@ -95,18 +99,52 @@ In my private registry:
   secret/regcred created
 </code></pre>
 
-You can also check the secret has been correctly deployed in your Kubernetes cluster:
+You can check the secret has been correctly deployed in your Kubernetes cluster:
+
+<pre class="console"><code>$ kubectl get secret regcred -o jsonpath="{.data.\.dockerconfigjson}"
+
+ewogICAgICAgICJhdXRocyI6IHsKCQkiaHR0cHM6Ly9pbmRleC5kb2NrZXIuaW8vdjEvIjogewogICAgICAgICAgICAgICAgICAgICAgICAiYXV0aCI6ICJjMk55WVd4NU9qaDBhM00wWm01aiIKICAgICAgICAgICAgICAgIH0sCiAgICAgICAgICAgICAgICAiY3g2ZHMzMGQuZ3JhNy5jb250YWluZXItcmVnaXN0cnkub3ZoLm5ldCI6IHsKICAgICAgICAgICAgICAgICAgICAgICAgImF1dGgiOiAiY0hKcGRtRjBaUzExYzJWeU9sQnlhWFpoZEdWVmMyVnlNUT09IgogICAgICAgICAgICAgICAgfQogICAgICAgIH0KfQo=
+</code></pre>
+
+>[!primary]
+>
+> Secrets are encoded in base64 and automatically decoded when they are attached to a Pod.
+
+### 2. Create a Secret by providing credentials on the command line
+
+Let's create a Secret of `docker-registry` type.
+
+You will use this Secret to authenticate with your private registry to pull a private image.
+
+```
+kubectl create secret docker-registry regcred \
+    --docker-server=<your-registry-server> \
+    --docker-username=<your-username> \
+    --docker-password=<your-password>
+```
+
+In my private registry:
```

Comment:
To keep consistency, maybe this would be better.
```suggestion
In your private registry:
```

Reply:
Here I'm showing an example from my demo registry